src/iplant_groups/service/subjects.clj

```diff
@@ -2,11 +2,35 @@
   (:require [iplant-groups.clients.grouper :as grouper]
             [iplant-groups.service.format :as fmt]))
 
+(defn get-creator-id
+  [group]
+  (get-in group [:detail :created_by]))
+
+(defn get-creator-ids
+  [groups]
+  (-> (mapv #(get-creator-id %) groups)
+      (distinct)))
+
+(defn match-subject-for-group
```

Comment:
Since these 3 functions are currently only used inside `iplant-groups.service.subjects`, I would mark them as private by using `defn-`.

This can be helpful in the future if the functions using these are refactored or removed later, then it's more obvious that these functions can be refactored/removed without affecting anything else outside this namespace.

Reply:
Ah, good point - will update!